src/adapters/github.js

```diff
@@ -76,17 +76,28 @@ class GitHub extends PjaxAdapter {
 
   // @override
   updateLayout(sidebarPinned, sidebarVisible, sidebarWidth) {
-    const SPACING = 10;
+    const SPACING = 20;
```

Comment:
Why not 10? Does it change existing behavior somehow?

Reply:
Yes it changes the spacing between sidebar and the content